pkg/operator/controller/names.go

```diff
@@ -15,26 +15,19 @@ const (
 	// CA certificate in this namespace.
 	GlobalMachineSpecifiedConfigNamespace = "openshift-config-managed"
 
-	// caCertSecretName is the name of the secret that holds the CA certificate
-	// that the operator will use to create default certificates for
-	// ingresscontrollers.
-	caCertSecretName = "router-ca"
-
-	// caCertConfigMapName is the name of the config map with the public key
-	// for the CA certificate, which the operator publishes for other
-	// operators to use.
-	caCertConfigMapName = "router-ca"
-
-	// routerCertsGlobalSecretName is the name of the secret with the
-	// default certificates and their keys, which the operator publishes for
-	// other operators to use.
-	routerCertsGlobalSecretName = "router-certs"
-
-	// controllerDeploymentLabel identifies a deployment as an ingress controller
+	// ControllerDeploymentLabel identifies a deployment as an ingress controller
 	// deployment, and the value is the name of the owning ingress controller.
-	controllerDeploymentLabel = "ingresscontroller.operator.openshift.io/deployment-ingresscontroller"
+	ControllerDeploymentLabel = "ingresscontroller.operator.openshift.io/deployment-ingresscontroller"
 )
 
+// IngressClusterOperatorName returns the namespaced name of the ClusterOperator
+// resource for the operator.
+func IngressClusterOperatorName() types.NamespacedName {
+	return types.NamespacedName{
+		Name: "ingress",
```

Comment:
Why use a string instead of a constant?

Reply:
Because it seems redundant to define a named constant that is only used to define (what is effectively) another named constant.